.golangci.yml

```
@@ -16,28 +16,29 @@ run:
   build-tags:
     - unit
```

```suggestion
    - unit
    - allcomponents
    - subtlecrypto
```

Missing build tags - as per workflow